controllers/discordactions.js

```diff
@@ -121,7 +120,38 @@ const addGroupRoleToMember = async (req, res) => {
     return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
   }
 };
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeNicknameOfUsers = async (req, res) => {
```

Comment:
Nit: we are changing a single user name at time. we can name the function better such as `changeUserNickname`